src/model/enums/country.rs

```diff
@@ -0,0 +1,525 @@
+use serde::{Deserialize, Serialize};
+use strum::{AsRefStr, Display, EnumString};
+
+///ISO 3166-1 alpha-2 country code, [wiki about ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+///Source from [country-list](https://datahub.io/core/country-list)
+#[derive(
+    Clone, Serialize, Deserialize, Copy, PartialEq, Eq, Debug, EnumString, AsRefStr, Display,
+)]
+pub enum Country {
```

Comment:
Isn't there a crate for this already? I would really prefer that if possible. It saves us 500 lines of untested and unmaintained code, compared to a well known crate. `aspotify` seems to use `isocode`: https://docs.rs/aspotify/0.6.1/aspotify/enum.CountryCode.html

Reply:
I don't think it's necessary to add a new crate just to take one `enum` from this crate, which will break the consistency of `enums` module, some enums are defined in `enums` module, and others are defined in another crate, adding such a new crate doesn't pay off.